colab-api/src/main/java/ch/colabproject/colab/api/setup/ColabConfiguration.java

```diff
@@ -140,6 +144,16 @@ public class ColabConfiguration {
      */
     public static final String LOCAL_ACCOUNT_BUTTON_VALUE = "true";
 
+    /**
+     * Date when the terms of service and user policy was last changed
+     */
+    public static final String TERMS_DATE = "colab.terms.date";
```

Comment:
rename TERMS_DATE to TERMS_OF_USE_DATE
and add a comment regarding expected formatting